docs/getting-started/devenv/get-started-with-wp-env.md

```diff
@@ -4,6 +4,8 @@ The [@wordpress/env](https://www.npmjs.com/package/@wordpress/env) package (`wp-
 
 Before following this guide, install [Node.js development tools](/docs/getting-started/devenv#node-js-development-tools) if you have not already done so.
 
+[![wp-env basics diagram](https://raw.githubusercontent.com/WordPress/gutenberg/HEAD/docs/assets/wp-env-diagram.png)](https://excalidraw.com/#json=8Tp55B-R6Z6-pNGtmenU6,_DeBR1IBxuHNIKPTVEaseA)
```

Comment:
I am wondering if the graphic would make more sense at the end of the Quick start section 🤔 That way the image reinforces the steps above it.

Reply:
I think it's more useful (and accessible) at the beginning as some sort of TL;DR